requirements.txt

```diff
@@ -26,5 +26,5 @@ six==1.11.0
 SQLAlchemy
 thespian==3.9.2
 Werkzeug
-git+https://github.com/vddesai1871/hydra-python-core@hydra_link#egg=hydra_python_core
+git+https://github.com/HTTP-APIs/hydra-python-core@develop#egg=hydra_python_core
```

Comment:
why we are not just merging hydra-python-core to master? @xadahiya @chrizandr

Reply:
Done

Reply:
@vddesai1871 please update this then we can merge

Reply:
Done